crowdin.yaml

```diff
@@ -5,8 +5,8 @@ preserve_hierarchy: true
 
 files:
   -
-    source: '/docs/*.md'
-    translation: '/website/translated_docs/%locale%/%original_file_name%'
+    source: '/docs/**/*.md'
+    translation: '/website/translated_docs/%locale%/**/%original_file_name%'
```

Comment:
Should the Crowdin example shown at https://docusaurus.io/docs/en/translation#crowdin be updated with this change?